packages/reassure-measure/src/measure.tsx

```diff
@@ -85,6 +78,20 @@ export async function measureRender(ui: React.ReactElement, options?: MeasureOpt
   return processRunResults(runResults, dropWorst);
 }
 
+export function buildUiToRender(
+  ui: React.ReactElement,
+  onRender: React.ProfilerOnRenderCallback,
+  Wrapper?: React.ComponentType<any>
+) {
+  const uiWithProfiler = (
+    <React.Profiler id="REASSURE_ROOT" onRender={onRender}>
+      {ui}
+    </React.Profiler>
+  );
+
+  return Wrapper ? <Wrapper>{uiWithProfiler}</Wrapper> : uiWithProfiler;
```

Comment:
how about making this an early return when `Wrapper` is not defined? We'd avoid creating the React.Profiler tree